### PR TITLE
Refactor PFColors

### PR DESCRIPTION
**Pull request by aljesusg**
Refactor PFColors. Only replace colors in one site

### CODE REVIEW

File: frontend/src/components/Pf/PfColors.tsx [hunk 1]
```diff
@@ -127,43 +131,54 @@ export type PFColorValues = {
 
 export let PFColorVals: PFColorValues;
 
+/*
+  Subtract color from var
```
Comment (jshaughn) [inline frontend/src/components/Pf/PfColors.tsx:135]:
```suggestion
  Extract color from var
```

Comment (aljesusg) [reply frontend/src/components/Pf/PfColors.tsx]: Shit I marked this like resolved I'll do the change thanks

**Review by ferhoyos — DISMISSED** [review]
LGTM, I like the idea to have only one source of truth in PFColors. 

@aljesusg just a suggestion if you can remove `BackgroundColor300` variable because it is not used in Kiali and this color is only applicable in dark mode:

From PF light theme palette:
image

**Comment by aljesusg** [comment]
> LGTM, I like the idea to have only one source of truth in PFColors.
> 
> @aljesusg just a suggestion if you can remove `BackgroundColor300` variable because it is not used in Kiali and this color is only applicable in dark mode:
> 
> From PF light theme palette: image

Let keep it because I did this PR to keep things clear and aligned with backstage and i am going to use that color

**Comment by ferhoyos** [comment]
Then it is ok if you are going to use it in backstage. Just take into account that this color does not have any effect in light theme ;-)

**Comment by jshaughn** [comment]
> i am going to use that color...

If there is a color you want to keep bit it's not used in the Kiali code base (and won't be found in a search) then I suggest a comment, otherwise someone (like me) may remove it.

**Review by ferhoyos — APPROVED** [review]
LGTM

**Pull request merged**